components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -168,6 +171,32 @@ open class GleanInternalAPI {
         }
     }
 
+    /**
+     * Records a list of the currently enabled accessibility services.
+     *
+     * https://developer.android.com/reference/android/view/accessibility/AccessibilityManager.html
+     */
+    private fun recordEnabledAccessibilityServices(applicationContext: Context) {
+        val accessibilityManager = applicationContext.getSystemService(
+            Context.ACCESSIBILITY_SERVICE
+        ) as AccessibilityManager
+        if (!accessibilityManager.isEnabled()) {
+            return
+        }
+        val serviceInfos: List<AccessibilityServiceInfo>
+        serviceInfos = accessibilityManager.getEnabledAccessibilityServiceList(
```

Comment:
Let's make this more Kotlin...ionic? We could use `map` or something along the lines of:

```
return accessibilityManager.getEnabledAccesibilityServiceList(...).map {
  it.loadDescription(packageManager)
}
```